scripts/vscripts/statcollection/init.lua

```diff
@@ -0,0 +1,20 @@
+require("statcollection/schema")
+require('statcollection/lib/statcollection')
+
+statInfo = LoadKeyValues('scripts/vscripts/statcollection/settings.kv')
```

Comment:
Why do you want these in the global space? The only times I see you wanting to use these would be when you have access to the library variables anyway.

Seems nit picky, but with generic names like these... with my limited understanding, I could see there being a chance of a dev overwriting them.

You may also want add a new option for "MIN_PLAYERS". There potentially could be mods that can work as single player games, of which they would want stats for.